app/controllers/posts_controller.rb

```diff
@@ -7,4 +8,21 @@ def index
   def show
     @post = Post.find(params[:id])
   end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.new(params[:id])
+    @post.title = params[:title]
+    @post.text = params[:text]
```

Comment:
**Optional**
- [ ] for code re-userbility, Kindly add a private method for the post_params